torch/csrc/autograd/functions/batch_normalization.cpp

```diff
@@ -17,6 +17,10 @@ namespace torch { namespace autograd {
 
 using thpp::Tensor;
 
+#ifndef CUDNN_BN_MIN_EPSILON
+#define CUDNN_BN_MIN_EPSILON 0
```

Comment:
According to https://github.com/pytorch/pytorch/issues/1623#issuecomment-303766744 , shouldn't it be something like `1e-5` or `1e-4` ?